src/annotator/test/guest-test.coffee

```diff
@@ -337,24 +330,14 @@ describe 'Guest', ->
       fakeSidebarFrame?.remove()
 
     it 'hides sidebar when the user taps or clicks in the page', ->
-
       for event in ['click', 'touchstart']
-        sandbox.spy(guest, methods[event])
```

Comment:
The removed lines in this file were testing implementation details of `Guest` which have changed in this PR.